cpplite/cpplite.debugger/src/org/netbeans/modules/cpplite/debugger/CPPLiteActionsProvider.java

```diff
@@ -69,7 +69,7 @@ public CPPLiteActionsProvider(ContextProvider contextProvider) {
     }
 
     @Override
-    public Set getActions () {
+    public Set<Object> getActions () {
```

Comment:
I strongly suggest not to do this. This overrides `getActions`  from `org.netbeans.spi.debugger.ActionsProvider`, which is not generified. Once it is, this might break. I doubt that `Object` is a valid generic for a method that should return Actions.